src/testing/mod.rs

```diff
@@ -8,7 +8,9 @@ mod consensus;
 mod crash;
 #[cfg(test)]
 mod dag;
-#[cfg(test)]
+#[cfg(feature = "fuzz")]
+pub mod fuzz;
```

Comment:
This whole setup is neither here nor there. Almost the whole implementation relevant to fuzzing is in the main crate, but the binaries themselves are separated. I would say either completely move it outside, just exporting some mock/test data types, so that it actually can be coded outside of the crate, or move the fuzz binaries into the crate. The advantage of the former is no `tokio` and other dependencies in alephbft (even if these were just conditional on "fuzz"), the advantage of the latter is no need to make any mock/test types `pub`. I feel like now we have about the worst of both worlds, but I'm not sure which is actually better.

Reply:
It seems like it shouldn't be part of the same package due to the way it is compile, otherwise it is messing with workspace/cargo. I moved it to a separate package.